node/pkg/driver/device_connectivity/device_path_s390x.go

```diff
@@ -0,0 +1,5 @@
+package device_connectivity
+
+// https://cgit.freedesktop.org/systemd/systemd/tree/src/udev/udev-builtin-path_id.c#n613
+// on Z-Systems, the udev subsystem is a Channel Command Word
+var FcSubsystem = "ccw"
```

Comment:
please add an ut to verify it.

Reply:
do you mean that the ut will have two consts with the same values, and it will choose which to compare to `FcSubsystem` according to the runtime architecture?

Reply:
done. hope it's OK